adminapp/src/pages/OrganizationMembershipForm.jsx

```diff
@@ -44,7 +44,6 @@ export default function OrganizationMembershipForm({
           helperText={orgText}
           value={resource.verifiedOrganization?.name}
           fullWidth
-          required
```

Comment:
Why isn't this required?

Reply:
@rgalanakis If we want to update other member data and save, the organization field will ask for an input, but it should be allowed to be left empty if we don't want to verify the organization membership.

Reply:
This isn't a sufficient fix- it just creates a new bug. We need to be able to modify the name and id separately with proper validation. Please file a bug so we can fix this properly, it should not be here.